app/src/forms/file/index.js

```diff
@@ -14,8 +14,5 @@ fileUpload.init({
 });
 
 module.exports.mount = (app) => {
-  const p = `/${_PATH}`;
-  app.use(p, routes);
-  app.use(dataErrors);
-  return p;
+  return setupMount(_PATH, app, routes);
```

Comment:
Changing the file/index to use the same `setupMount` utility function that all the other index files use.